build/common.project.props

```diff
@@ -169,6 +168,11 @@
     </AssemblyAttributes>
   </ItemGroup>
 
+  <!-- Assembly attributes for net core projects -->
+  <PropertyGroup Condition=" '$(IsNetCoreProject)' == 'true' ">
+    <AssemblyVersion>$(SemanticVersion).$(PreReleaseVersion)</AssemblyVersion>
```

Comment:
Why do you need to set this explicitly? Having `$(Version)` should be enough to make msbuild set correct assembly version.